### PR TITLE
Istio update 1.14.3

### PR DESCRIPTION
**Pull request by mitulshah-suse**
Related issue https://github.com/rancher/rancher/issues/38451
https://github.com/rancher/rancher/issues/38968
additional PRs opened 
https://github.com/rancher/image-mirror/pull/275
https://github.com/rancher/istio-installer/pull/59

### CODE REVIEW

File: packages/rancher-istio/1.14/rancher-kiali-server/generated-changes/patch/templates/deployment.yaml.patch [hunk 1]
```diff
@@ -32,7 +32,7 @@
 +          items:
 +          - key: env.js
 +            path: env.js
-+      {{- end }}
++      {{- end }}
```
Comment (doflamingo721) [inline packages/rancher-istio/1.14/rancher-kiali-server/generated-changes/patch/templates/deployment.yaml.patch:35]: Nit: @mitulshah-suse Can you please remove the whitespaces after `{{- end}}`.

File: packages/rancher-istio/1.14/rancher-kiali-server/generated-changes/patch/templates/deployment.yaml.patch [hunk 2]
```diff
@@ -17,7 +17,7 @@
 +        - name: kiali-console
 +          subPath: env.js
 +          mountPath: /opt/kiali/console/env.js
-+        {{- end }}
++        {{- end }}
```
Comment (doflamingo721) [inline packages/rancher-istio/1.14/rancher-kiali-server/generated-changes/patch/templates/deployment.yaml.patch:20]: Nit: Can you please remove the whitespaces after `{{- end}}`.

**Review by doflamingo721 — DISMISSED** [review]
lgtm

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-istio/1.14/rancher-kiali-server/package.yaml [hunk 3]
```diff
@@ -1,3 +1,3 @@
-url: https://kiali.org/helm-charts/kiali-server-1.52.0.tgz
+url: https://kiali.org/helm-charts/kiali-server-1.55.0.tgz
 version: 100.0.0
```
[the remark this answers is not in the document]
Comment (mitulshah-suse) [reply packages/rancher-istio/1.14/rancher-kiali-server/package.yaml]: Updated

File: packages/rancher-istio/1.14/rancher-tracing/charts/values.yaml [hunk 4]
```diff
@@ -28,7 +28,7 @@ global:
 
 jaeger:
   repository: rancher/mirrored-jaegertracing-all-in-one
-  tag: 1.35.1
+  tag: 1.37.0
```
[the remark this answers is not in the document]
Comment (mitulshah-suse) [reply packages/rancher-istio/1.14/rancher-tracing/charts/values.yaml:31]: Updated

**Comment by eliyamlevy** [comment]
Merge issues

**Review by aiyengar2 — DISMISSED** [review]

**Review by doflamingo721 — APPROVED** [review]

**Pull request merged**